assets/src/edit-story/components/header/buttons.js

```diff
@@ -183,8 +184,12 @@ function Publish() {
   const text = hasFutureDate
     ? __('Schedule', 'web-stories')
     : __('Publish', 'web-stories');
+
   return (
-    <Primary onClick={handlePublish} isDisabled={isSaving || isUploading}>
+    <Primary
```

Comment:
I suppose it's a UX question but should this be displayed at all without the correct permissions?

If we show anything then maybe we would need to bring in the `pending` status and use the "Submit for Review" option as in WordPress. This would need to be confirmed by UX though.

I'm just wondering how to make sure the "Publish" button doesn't feel broken if it's disabled due to a lack of permissions.

Maybe keep as is but follow up with UX? WDYT?